### PR TITLE
wheel change event

### PR DESCRIPTION
**Pull request by gnawalishyam**

### CODE REVIEW

**Review by Lahasyakr — COMMENTED** [review]
do we need this?
we need ctrl + wheel  for zoom

File: PROTOTYPE6/src/js/RealTimeDrawer.js [hunk 1]
```diff
@@ -40,7 +41,14 @@ RealTimeDrawer.prototype.setUpInteraction = function () {
...
 
 }
-
+RealTimeDrawer.prototype.onWheel = function (m) {
+    var u = this.nv.canvas.getBoundingClientRect();
+    var data = {
+        "mouse_args": { "deltaY": m.deltaY, "clientX": m.clientX, "clientY": m.clientY },
+        "canvas_info": { "left": u.left, "top": u.top }
+    }
+    LINK.trigger('client-receive-wheel', { "data": data });
```
Comment (gnawalishyam) [inline PROTOTYPE6/src/js/RealTimeDrawer.js:50]: should we put if (key code is ctrl key) above line no. 50?

Comment (gnawalishyam) [reply PROTOTYPE6/src/js/RealTimeDrawer.js:50]: what should we do if there is only wheel?

Comment (Lahasyakr) [reply PROTOTYPE6/src/js/RealTimeDrawer.js:50]: i was working on Zoom, and I found out that changing drag modes on the ctrl + wheel is not working properly
but it works firn with just wheels.
so if that's the case I don't know how exactly to accommodate this

Comment (Lahasyakr) [reply PROTOTYPE6/src/js/RealTimeDrawer.js:50]: when we change drag modes, wheel works differently

Comment (gnawalishyam) [reply PROTOTYPE6/src/js/RealTimeDrawer.js:50]: if we already finished 2 features that uses wheel, then we can report professor to understand if one or both are useful. are you putting some key combination or is it just wheel change event like mine? is zoom working differently than mine?

Comment (Lahasyakr) [reply PROTOTYPE6/src/js/RealTimeDrawer.js:50]: yours is zoom?

Comment (Lahasyakr) [reply PROTOTYPE6/src/js/RealTimeDrawer.js:50]: i dint work like zoom to me, it just changed some structures

**Pull request merged**